src/pages/register.tsx

```diff
@@ -1,91 +1,107 @@
 import { useContext, useState } from "react";
-
 import { useRouter } from "next/router";
-
 import { Dayjs } from "dayjs";
 
 import { userContext } from "supabase/user_context";
 import { useSupabaseClient } from "@supabase/auth-helpers-react";
 import { Database } from "supabase/db_types";
-
-import { Box, Stack, Button, SelectChangeEvent, Grid } from "@mui/material";
+import { Box, Stack, Button, SelectChangeEvent, Grid, Card, Typography } from "@mui/material";
+import { grey } from "@mui/material/colors";
 
 import Loading from "@/components/public/Loading";
+import Background from "@/components/public/Background";
 import Logo from "@/components/public/Logo";
 import CommonTextField from "@/components/public/CommonTextField";
 import PasswordTextFeild from "@/components/public/PasswordTextField";
-import CommonDropdown from "@/components/public/CommonDropdown";
+import CommonDropdown from "@/components/public/GenderDropdown";
 import CommonDatePicker from "@/components/public/CommonDatePicker";
-
 import { validateEmail, validateTextField } from "@/utilities/validation";
+
 import { validation } from "@/types/Validation";
-import { Gender } from "enum/gender";
-import { CHAR_LIMIT } from "enum/inputLimit";
-import { PagePaths } from "enum/pages";
+import { GENDER } from "enum/GENDER";
```

Comment:
Enum Error

Reply:
Solved

Reply:
In my file, there's still an error.
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/62459518/226829094-1f16ff8d-d28f-4152-be4c-87952dd9a142.png">